EIPS/eip-145.md

```diff
@@ -27,51 +27,51 @@ The following instructions are introduced:
 
 ### `0x1b`: `SHL` (shift left)
 
-The `SHL` instruction (shift left) pops 2 values from the stack, `arg1` and `arg2`, and pushes on the stack the first popped value `arg1` shifted to the left by the number of bits in the second popped value `arg2`. The result is equal to
+The `SHL` instruction (shift left) pops 2 values from the stack, `arg1` and `arg2`, and pushes on the stack the second popped value `arg2` shifted to the left by the number of bits in the first popped value `arg1`. The result is equal to
```

Comment:
I would skip "second popped" / "first popped". If the order is not clear, it should be clarified in the first part of the sentence.

How about
> ... pushes on the stack the result of the `arg2` shifted to the left by the number of bits in `arg1`.